macros/math/specialTrigValues.pl

```diff
@@ -106,7 +107,7 @@ sub specialAngle {
 	} else {
 		$a = $closest->[0];
 	}
-	my $return = Formula("$a pi $divc");
+	my $return = ($a ne '0') ? Formula("$a pi $divc") : Formula("0");
```

Comment:
You can just use single quotes here, `Formula('0')`.

Reply:
OK, I changed that.

Reply:
OK, single quotes in the POD now too.

Reply:
Oh, I always overlook this too, but `Formula` already treats the input as a string, so `Formula(0)` would have been sufficient. I'm unsure out of what it looks like in the code if there is any advantage here.